controls/radgridview/rows/selected-row-background.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Selected Row Background
+page_title: Selected Row Background
+description: Selected Row Background
+slug: radgridview-rows-selected-background
+tags: selected,row,background
+published: True
+position: 4
+---
+
+# Selected Row Background
+
+__RadGridView__ has a default __Brush__ for its selection on row level depending on the given theme. For example, the visual appearance of the selected row in the __Office2016__ theme would be as shown below.
+
+#### __Figure 1: Selected Row Background in Office2016 Theme__
+
+![Selected Row Background in Office2016 Theme](images/RadGridView_SelectedRow_Background_01.png)
+
+As of __R3 2018 RadGridView__ provides support for setting the __Background__ of its selected rows through the __SelectedBackground__ property of __GridViewRow__.

```

Comment:
I think here we can remove the bolding from the word Background and also start it lower case letter "background" because we are not talking about a property or a class name.